localization/english/event_localization/wc_draenor_events_l_english.yml

```diff
@@ -23,3 +23,13 @@
  WCDRA_25_TITLE:0 "The Dark Portal Reopened"
  WCDRA_25_DESC:0 "WRITE ME"
  WCDRA_25_OPT_A:0 "WRITE ME"
+
+ WCDRA_200_TITLE:0 "Intruders Beyond the [ROOT.Char.Custom('GetPortalRift')]"
+ WCDRA_200_DESC_beginning:0 "Your soldiers report they faced armed forces coming out of the [ROOT.Char.Custom('GetDarkPortalRift')]. So far their numbers are unknown, but it's clear there is some movement beyond the veil.\n\nRumors say these aliens are"
+ WCDRA_200_DESC_draenei:0 "draenei survivors led by a powerful exarch blessed by the naaru."
+ WCDRA_200_DESC_arakkoa:0 "arakkoa outcasts led by a reincarnation of Terokk, the greatest king."
+ WCDRA_200_DESC_orcs:0 "members of wild and despair orcish clans gathered together."
+ WCDRA_200_DESC_ogres:0 "members of wild and despair ogre clans gathered together."
```

Comment:
Change the wording:
"members of wild and disparate ogre clans gathered together."